bcipy/signal/tests/model/test_model.py

```diff
@@ -56,30 +56,32 @@ def setUp(self):
         baseline_dir=expected_output_folder, filename="test_inference.expected.png", remove_text=True
     )
     def test_inference(self):
+        np.random.seed(0)
```

Comment:
Would it make sense to add `np.random.seed(0)` to the end of the `setup` method?